Jellyfin.Api/Controllers/UserLibraryController.cs

```diff
@@ -1,14 +1,18 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using System.IO;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Jellyfin.Api.Constants;
 using Jellyfin.Api.Extensions;
+using Jellyfin.Api.Helpers;
 using Jellyfin.Api.ModelBinders;
+using Jellyfin.Api.Models.UserDtos;
 using Jellyfin.Data.Enums;
 using Jellyfin.Extensions;
+using Kfstorm.LrcParser;
```

Comment:
Is there a specific reason this is precompiled?

Reply:
The one straight from NuGet targets the wrong TargetFramework. I recompiled it with a modified TargetFramework value of `net6.0`

Reply:
I thought I left a comment- we straight up cannot use this library as it is unlicensed. We also dont directly include dependencies.

Reply:
I've found a different NuGet package to use for the LRC parser that has an MIT license.